https://dev.gnupg.org/rG9ae3cfcabec9252c22d67b7a15c36f0a8cf22f0f
dirmngr: Enable the call of ks_ldap_help_variables when USE_LDAP.
9ae3cfcabec9

Index: dirmngr/server.c
--- dirmngr/server.c.orig
+++ dirmngr/server.c
@@ -2776,7 +2776,9 @@ cmd_ad_query (assuan_context_t ctx, char *line)
 
   if (opt_help)
     {
+#if USE_LDAP
       ks_ldap_help_variables (ctrl);
+#endif
       err = 0;
       goto leave;
     }
